CMakeLists.txt

```diff
@@ -219,6 +219,17 @@ target_link_libraries(${NVFUSER_CODEGEN} PRIVATE stdc++fs)
 target_link_libraries(${NVFUSER_CODEGEN} PRIVATE ${CUDA_NVRTC_LIB} ${LIBNVTOOLSEXT})
 target_include_directories(${NVFUSER_CODEGEN} PRIVATE ${CUDA_INCLUDE_DIRS})
 
+if (NOT UCC_FOUND)
+  # This probably means a "standalone" build, where UCC_FOUND is not inherited from pytorch's CMake configuration.
+  # It's required to set env vars UCC_DIR, UCX_DIR, UCC_HOME, UCX_HOME for CMake's find UCC to work.
+  find_package(UCC REQUIRED)
+  find_package(UCX REQUIRED)
```

Comment:
Looks like CI node doesn't have ucc installed.

Looking at Xiang's PR, there seems to be a script doing that already: https://github.com/pytorch/pytorch/commit/cda210e23b6bb42658e7e4f70778f9db7a900645#diff-a45bde6f6abdaabff96d1742357b0817fd3e3e98dd7e534a744fb955690d2ba3

.circleci/docker/common/install_ucc.sh

Reply:
UCC is installed by default in NGC cuda container

Reply:
I meant the standalone build ci using github nodes: https://github.com/NVIDIA/Fuser/blob/41f4dce9c8e97f811ba5704b7a69c56061ab8672/.github/workflows/lint.yml#L13-L29